src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin.StrongNamed/DefaultOmniSharpProjectSnapshotManagerDispatcher.cs

```diff
@@ -25,6 +26,11 @@ private class OmniSharpProjectSnapshotManagerDispatcher : ProjectSnapshotManager
             public OmniSharpProjectSnapshotManagerDispatcher() : base(ThreadName)
             {
             }
+
+            public override void LogException(Exception ex)
+            {
+                // ActivityLog and the LSP logger aren't applicable to O#. Do nothing.
```

Comment:
Is this an accurate comment? I wasn't sure what to do in the O# case.

Reply:
It'd probably be more accurate to say we don't have a good mechanism currently for grabbing data from the O# side. They probably expose a logger somewhere but it's not a priority to hook all that together yet 😄